readme.md

```diff
@@ -34,26 +34,63 @@ A starter app is available on [codesandbox](https://codesandbox.io/s/github/matt
 ```shell
 npm install haunted
 ```
+
 For Internet Explorer 11, you'll need to use a proxy polyfill, in addition to the usual webcomponentsjs polyfills. 
+
 eg.
 ```html
-<script type="text/javascript" src="https://cdn.jsdelivr.net/npm/proxy-polyfill@0.3.0/proxy.min.js"></script> 
+<script src="https://cdn.jsdelivr.net/npm/proxy-polyfill@0.3.0/proxy.min.js"></script> 
 ```
-For a full example see - https://github.com/crisward/haunted-ie11
+
+For a full example with Internet Explorer 11, see - https://github.com/crisward/haunted-ie11
 
 You can also use Custom Elements without Shadow DOM if you wish.
 eg.
+
+```js
+component(() => html`...`, HTMLElement, { useShadowDOM: false }));
 ```
-component(() => html`...`, HTMLElement, {useShadowDOM: false}))
+
+### Importing
+
+__Haunted__ can be imported just like any other library when using a bundler of your choice:
+
+```js
+import { component, html, useState } from 'haunted';
 ```
 
-### Builds
+The main entry point is intended for [lit-html](https://github.com/Polymer/lit-html) users.
+
+#### lighterhtml, etc
+
+If you are using [lighterhtml](https://github.com/WebReflection/lighterhtml) or [hyperHTML](https://github.com/WebReflection/hyperHTML) then instead import `haunted/core`. This export gives you a function that creates Hooks that work with any template library.
+
+```js
+import haunted, { useState } from 'haunted/core';
+import { html, render } from 'lighterhtml';
 
-Haunted comes in a few builds. Pick one based on your chosen environment:
+const { component } = haunted({
+  render(what, where) {
+    render(where, () => what);
+  }
+});
+
+const App = component(() => {
+  const [count, setCount] = useState(0);
+  return html`Using lighterhtml! Count: ${count}`;
+});
+```
+
+#### Web modules
+
+__Haunted__ can work directly in the browser without using any build tools. Simply import the `haunted.js` bundle. You can use the [unpkg] CDN. This works great for demo pages and small apps:
+
+```js
+import { html } from 'https://unpkg.com/lit-html/lit-html.js';
+import { component, useState, useEffect } from 'https://unpkg.com/haunted/haunted.js';
+```
 
-* __index.js__ is available for bundlers such as Webpack and Rollup. Use with: `import { useState } from 'haunted';`;
-* __web.js__ is avaible for use with the web's native module support. Use with: `import { useState } from '../node_modules/haunted/web.js';`.
-* __haunted.js__ is available via the CDN [unpkg](https://unpkg.com). This is great for small apps or for local development without having to install anything. Use with `import { useState } from 'https://unpkg.com/haunted/haunted.js';`
+If you install Haunted locally this build is located at `node_modules/haunted/haunted.js`.
```

Comment:
i'd love it if we could add the pika bundle alternative too

```js
import { html } from "https://cdn.pika.dev/lit-html"
import { component, useState, useEffect } from "https://cdn.pika.dev/haunted"
```

Reply:
Happy to do that